### PR TITLE
1684 format with prettier

### PR DESCRIPTION
**Pull request by prupipho**
## PR checklist

- Quick summary of changes
run prettier for .js, .ts, .tsx files on solution to reduce complex PR due to formatting changes

- Which issue does this PR relate to?
#1684

### CODE REVIEW

File: src/client/src/components/Modal/index.tsx [hunk 1]
```diff
@@ -64,26 +63,22 @@ interface IProps {
...
-
         <Modal
           isOpen={this.props.isModalOpen}
           onRequestClose={this.props.closeModal}
           contentLabel="Modal Display"
           style={getCustomStyles(MODAL_TYPE)}
           ariaHideApp={false}
         >
-          <WrappedComponent {...this.props as P} />
+          <WrappedComponent {...(this.props as P)} />
```
Comment (prupipho) [inline src/client/src/components/Modal/index.tsx:81]: review this

File: src/client/src/components/Modal/index.tsx [hunk 2]
```diff
@@ -64,26 +63,22 @@ interface IProps {
...
-const asModal = <P extends object>(
-  WrappedComponent: React.ComponentType<P>,
-  MODAL_TYPE?: ModalType
-) => {
+const asModal = <P extends object>(WrappedComponent: React.ComponentType<P>, MODAL_TYPE?: ModalType) => {
   return class extends React.Component<P & IProps> {
     static defaultProps = {
-      closeModal: () => void(0),
-      isModalOpen: false
+      closeModal: () => void 0,
```
Comment (prupipho) [inline src/client/src/components/Modal/index.tsx:69]: review this

File: src/client/.eslintrc.js [hunk 3]
```diff
@@ -1,50 +1,41 @@
...
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/interface-name-prefix": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
+    "@typescript-eslint/no-namespace": "off",
+    "@typescript-eslint/no-var-requires": "off",
+    "react/display-name": "off",
+    "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/no-use-before-define": "off",
+    "react-hooks/exhaustive-deps": "off",
+    eqeqeq: "warn",
```
Comment (prupipho) [inline src/client/.eslintrc.js:30]: test this

File: src/client/src/components/Modal/index.tsx (hunk 1 above)
Comment (prupipho) [reply src/client/src/components/Modal/index.tsx:81]: seems ok

**Review by sibille — APPROVED** [review]

**Pull request merged**